src/main/java/com/synopsys/integration/detect/lifecycle/run/DetectRun.java

```diff
@@ -80,6 +94,20 @@ public void run(BootSingletons bootSingletons) {
             } else {
                 bdio = BdioResult.none();
             }
+
+            if(autonomousManager.getAutonomousScanEnabled()) {
+                SortedMap<String, SortedSet<String>> packageManagerTargets = stepRunner.getScanTargets(universalToolsResult);
+                autonomousManager.updateScanTargets(packageManagerTargets, scanTypeEvidenceMap);
```

Comment:
Updating all the scan type targets in the file.

Reply:
Did you see any performance impact with SortedMap?

Reply:
No, as of now, did not see any performance impact for these changes.